health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/AppHelpTutorial.js

```
@@ -0,0 +1,205 @@
+const checkCurrentScreenVisible = (config, pathVar) => {
+  const listOfEnabledScreens = config?.flat();
+  return listOfEnabledScreens?.includes(pathVar);
+};
+const AppHelpTutorial = ({ appPath, location }) => {
+  const { t } = useTranslation();
+  const [visible, setVisible] = useState(true);
+  const [closing, setClosing] = useState(false);
+
+  const module = appPath?.split?.("/")?.[appPath?.split("/")?.length - 1];
+
+  const pathVar = location.pathname.replace(appPath + "/", "").split("?")?.[0];
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)